lib/plugins/aws/package/compile/events/websockets/lib/stage.js

```diff
@@ -60,14 +61,10 @@ module.exports = {
 
       Object.assign(cfTemplate.Resources, {
         [logGroupLogicalId]: getLogGroupResource(service, stage),
-        [logsRoleLogicalId]: getIamRoleResource(service, stage),
-        [accountLogicalid]: getAccountResource(logsRoleLogicalId),
       });
-    }
 
-    Object.assign(cfTemplate.Resources, { [stageLogicalId]: stageResource });
-
-    return BbPromise.resolve();
+      return ensureApiGatewayCloudWatchRole(this);
```

Comment:
We should add a test that this function was called.

Reply:
Added (and it indeed it uncovered one issue).

To make it reliable I needed to refactor `addCustomResourceToService` so it works in context of AWS provider and not `AwsCompileXEvents` instance